Sources/Rendering/OpenGL/RenderWindow/index.d.ts

```diff
@@ -241,20 +241,20 @@ export interface vtkOpenGLRenderWindow extends vtkOpenGLRenderWindowBase {
 
 	/**
 	 *
-	 * @param vtktype
-	 * @param numComps
-	 * @param useFloat
+	 * @param {VtkDataTypes} vtktype 
```

Comment:
Please chomp the trailing spaces.

Reply:
We will need to get the linter to format d.ts files. I don't think it does presently.

Reply:
The trailing space is needed, it indicates the description of the param it's just we don't it everywhere
* @param {paramType} paramName paramDescription

Reply:
Just to follow-up: I'm pretty sure if I let prettier run on d.ts files, it'll chomp trailing whitespace. I don't know if there is any special setting to tell prettier to avoid docstring whitespace.